### PR TITLE
added ES6 installation

### PR DESCRIPTION
**Pull request by mesqueeb**

### CODE REVIEW

**Comment by mesqueeb** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -89,6 +89,8 @@ You can then `require` the following modules within your source files:
 ```javascript
 var firebase = require('firebase');
 var firebaseui = require('firebaseui');
+// or
```
Comment (bojeil-google) [inline README.md]: // or for ES6 imports.

**Comment by mesqueeb** [comment]
any update on this PR?

**Comment by bojeil-google** [comment]
Comment change was requested.

**Comment by mesqueeb** [comment]
Now I understand you! I think I did it right : )

**Comment by mesqueeb** [comment]
@bojeil-google How can I "sync with HEAD"?

**Comment by bojeil-google** [comment]
Your branch was out of sync. Nevermind, looks like you fixed that.

**Pull request merged**